pkg/insights/insightsclient/insightsclient.go

```diff
@@ -66,6 +70,32 @@ func New(client *http.Client, maxBytes int64, metricsName string, authorizer Aut
 	}
 }
 
+func (c *Client) clientTransport() http.RoundTripper {
+	// default transport, proxy from env
+	clientTransport := &http.Transport{
+		Proxy: http.ProxyFromEnvironment,
```

Comment:
This may not work correctly for CIDRs in NO_PROXY when built with Go < 1.11, not sure if that could be a problem here, but it should be possible to avoid that with `k8s.io/apimachinery/pkg/util/net.NewProxierWithNoProxyCIDR()`

Can be addressed in a follow up if the above's a problem.